packages/web-components/src/components/card/card.ts

```diff
@@ -65,8 +65,12 @@ class DDSCard extends StableSelectorMixin(BXLink) {
   protected _handleSlotChange({ target }: Event) {
     const { name } = target as HTMLSlotElement;
     const hasContent = Boolean(this.querySelector('p'));
+    const hasPictogram = Boolean(
+      (target as HTMLSlotElement).assignedNodes().length
+    );
     this[slotExistencePropertyNames[name]] = hasContent;
     this._hasCopy = hasContent;
+    this._hasPictogram = hasPictogram;
```

Comment:
I'm finding that these changes are triggering an infinite rendering loop somehow and the browser becomes unresponsive. It's affecting any story that has a card in it 😕 . I'm pretty sure the culprit is here and has something to do with the fact that `_handleSlotChange` will fire for any slot changes amongst the default slot, `pictogram` slot, or `image` slot. There is always an initial `slotchange` event for every slot, and `hasPictogram` is being adjusted even when the slot change event is for `image` or the default slot. I found that adjusting things back to what they were and then appending `pictogram` specific code as follows fixes things:

```js
if (name === 'pictogram') {
  this._hasPictogram = Boolean(
    (target as HTMLSlotElement).assignedNodes().length
  );
}
```

For bonus points, I'd go further and suggest that we shouldn't have a shared `_handleSlotChange` event handler. There should be one per `<slot>` and it should be solely concerned with its relevant state variable (eg. `_hasPictogram` for the `pictogram` slot). I'd also get rid of the `slotExistencePropertyNames` abstraction, as it's hard to follow.